src/main/java/com/example/youthcon/SseEmitters.java

```diff
@@ -0,0 +1,147 @@
+package com.example.youthcon;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+
+@Component
+@Slf4j
+public class SseEmitters {
+
+    // todo(hun): 아티클별 조회수
+    private static final Map<String, AtomicLong> articleId2AtomicLong = new ConcurrentHashMap<>();
+
+    private final LoadingCache<String, SseEmitter> tabId2SseEmitter = CacheBuilder.newBuilder()
+                                                                    .expireAfterAccess(10000L, TimeUnit.MILLISECONDS)
+                                                                    .build(
+                                                                            new CacheLoader<String, SseEmitter>() {
+                                                                                @Override
+                                                                                public SseEmitter load(String key) {
+                                                                                    // 캐시 항목이 없거나 만료된 경우에 값을 로드하는 코드
+                                                                                    return null;
+                                                                                }
+                                                                            }
```

Comment:
```suggestion
    private final Cache<String, SseEmitter> tabId2SseEmitter = CacheBuilder.newBuilder()
                                                                    .expireAfterAccess(10000L, TimeUnit.MILLISECONDS)
                                                                    .build(
```

CacheLoader의 load 설명을 보면 null을 리턴하지 말라고 되어있습니다. 이 기능이 기본값을 넣어주는 기능이기 때문에 기본값을 넣지 않을 거라면 사용하지 않아도 될 것 같아요.

Reply:
넵 수정했습니다!